src/OpenTelemetry.Instrumentation.Wcf/TelemetryServiceBehavior.cs

```diff
@@ -14,9 +14,11 @@
 // limitations under the License.
 // </copyright>
 
+#pragma warning disable IDE0005 // Using directive is unnecessary.
```

Comment:
Why are we disabling the rule for this specific instance?

Reply:
I tried all combinations of removing these "unnecessary" using statements, but in each scenario VS would then complain and tell me they were needed. I wasn't sure what to do so disabled via source for this one spot.